ztp/gitops-subscriptions/argocd/example/acmpolicygenerator/acm-example-multinode-site.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: PolicyGenerator
+metadata:
+  name: example-multinode-site-latest
+placementBindingDefaults:
+  name: example-multinode-site-placement-binding
+policyDefaults:
+  # categories: []
+  #controls:
+  #  - PR.DS-1 Data-at-rest
+  namespace: ztp-site
+  # Use an existing placement rule so that placement bindings can be consolidated
+  placement:
+    labelSelector:
+      # These policies will correspond to all clusters with this label:
+      site: "example-multinode"
+      # the below label (du-profile: "latest") might not necassery as the site policy applied to a single site.
```

Comment:
I agree with the comment, but for consistency I like that we have this label here. I would recommend removing the comment.